src/Lykke.Service.HFT/Controllers/OrdersController.cs

```diff
@@ -44,24 +46,29 @@ public class OrdersController : Controller
         /// <returns>Client orders.</returns>
         [HttpGet]
         [SwaggerOperation("GetOrders")]
-        [ProducesResponseType(typeof(IEnumerable<LimitOrderState>), (int)HttpStatusCode.OK)]
-        public async Task<IActionResult> GetOrders([FromQuery] OrderStatus? status = null)
+        [ProducesResponseType(typeof(IEnumerable<Models.LimitOrderState>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> GetOrders([FromQuery] OrderStatus? status = null, [FromQuery] uint? skip = 0, [FromQuery] uint? take = 100)
         {
+            if (take > MaxPageSize)
+                return BadRequest(ResponseModel.CreateInvalidFieldError("take", $"Page size {take} is to big"));
+
             var clientId = User.GetUserId();
             var orders = status.HasValue
-                ? _orderStateRepository.FilterBy(x => x.ClientId == clientId && x.Status == status.Value)
-                : _orderStateRepository.FilterBy(x => x.ClientId == clientId);
-            return Ok(orders.OrderByDescending(x => x.CreatedAt));
+                ? _orderStateRepository.All().Where(x => x.ClientId == clientId && x.Status == status.Value)
```

Comment:
_orderStateRepository.All()
can there be a lot of orders in the database
so that this query is not effective, we are the same all the table unload?